helix-front/package.json

```diff
@@ -67,8 +68,8 @@
     "rxjs": "^6.5.5",
     "rxjs-compat": "^6.0.0-rc.0",
     "sass": "^1.51.0",
-    "tsickle": "^0.38.1",
-    "tslib": "^1.10.0",
+    "tsickle": "github:angular/tsickle#fdf6e86717648f1b9603752cb83486fa6dcac38f",
```

Comment:
This is necessary because tsickle did not publish a release which supports `"typescript": "4.0.8"` which we need for other packages.

Details in this Github issue comment: https://github.com/apache/helix/issues/2095#issuecomment-1128148592